docs/manual/output.md

```diff
@@ -91,3 +88,212 @@ expressed in the typical physical unit symbols. Unfortunately, column names
 cannot contain forward slashes, so units lkike `m/s` are not possible to
 represent directly. Instead, a backslash (`\`) is used to encode the division
 symbol (for example: `velocity_in_m\s`).
+
+## Germanium (HPGe) detectors
+
+The Germanium output scheme handles the output from Germanium (HPGe) detectors, but would
+also work for other solid state detectors (calorimeters).
+
+Germanium detectors have sensitivity to the topology of event interactions via the pulse
+shape and they also have a different response close to the detector electrodes. So
+when simulating HPGe's it is advisable to save the information of the steps of
+particles within the detector. Then "post-processing" software such as _reboost_
+([docs](https://reboost.readthedocs.io/en/stable/)) can apply the detector response
+model without repeating the computationally intensive simulation.
+
+By default this output scheme writes out all steps in the registered sensitive
+HPGe detectors. The following properties of each hit are recorded (by default):
+
+- _time_: The global time of the hit,
+- _particle_: the PDG code of the particle,
+- _xloc_, _yloc_, _zloc_: the global position,
+- _evtid_ : the index of the Geant4 event,
+- _edep_: the deposited energy,
+- _dist_to_surf_: the distance of the hit from the detector surface.
+
+By default all floating point fields are saved with 64-bit (double) precision.
+The precision of the energy and or position / distance fields can be reduced
+to 32-bit with the macro commands:
+
+`/RMG/Output/Germanium/StoreSinglePrecisionPosition true`
+
+and,
+
+`/RMG/Output/Germanium/StoreSinglePrecisionEnergy true`
+
+It is possible to also store the track id and parent track id of each step with:
+
+`/RMG/Output/Germanium/StoreTrackID true`
+
+As mentioned earlier output schemes also provide a mechanism for filtering
+events. One useful option is to only write out events in which energy
+was deposited in a Germanium detector. This is used since the
```

Comment:
Is Germanium capital because it's the name of the sensitive detector type? Then maybe use monospaced or italic font?